erpnext_thailand/thai_tax/report/sales_tax_report/sales_tax_report.html

```diff
@@ -0,0 +1,319 @@
+            
+                    {% var grand_tax_base = grand_tax_base + row.tax_base; %}
+                    {% var grand_tax_amount = grand_tax_amount + row.tax_amount; %}
+                    {% var grand_total = grand_total + row.tax_base + row.tax_amount; %}
+            
+                    {% if page_idx == 10 || idx == data.length - 1 %}
+                        <tr>
+                            <td class="text-center" colspan="8" style="background-color: #d3d3d3;"><strong>รวมหน้า</strong></td>
+                            <td class="text-right"><strong>{%= sum_tax_base.toLocaleString(undefined, {minimumFractionDigits: 2, maximumFractionDigits: 2}) %}</strong></td>
+                            <td class="text-right"><strong>{%= sum_tax_amount.toLocaleString(undefined, {minimumFractionDigits: 2, maximumFractionDigits: 2}) %}</strong></td>
+                            <td class="text-right"><strong>{%= sum_total.toLocaleString(undefined, {minimumFractionDigits: 2, maximumFractionDigits: 2}) %}</strong></td>
+                        </tr>
+            
+                        {% if idx == data.length - 1 %}
+                            <tr>
+                                <td class="text-center" colspan="8" style="background-color: #d3d3d3;"><strong>รวมทั้งหมด</strong></td>
+                                <td class="text-right"><strong>{%= grand_tax_base.toLocaleString(undefined, {minimumFractionDigits: 2, maximumFractionDigits: 2}) %}</strong></td>
+                                    <td class="text-right"><strong>{%= grand_tax_amount.toLocaleString(undefined, {minimumFractionDigits: 2, maximumFractionDigits: 2}) %}</strong></td>
+                                    <td class="text-right"><strong>{%= grand_total.toLocaleString(undefined, {minimumFractionDigits: 2, maximumFractionDigits: 2}) %}</strong></td>
+                            </tr>
+                        {% endif %}
+            
+                        {% var page_idx = 0; %}
+                        {% var sum_tax_base = 0; %}
+                        {% var sum_tax_amount = 0; %}
+                        {% var sum_total = 0; %}
+                    {% endif %}
+            
+                    {% var page_idx = page_idx + 1; %}
+                    {% endif %}
+                    {% var idx = idx + 1; %}
+                {% endfor %}
+            </tbody>
+        </table>
+    </div> 
+</div> 
+</body>
+<script>
+    document.addEventListener("DOMContentLoaded", function() {
+        let pageNumber = 1; 
+        let rowsPerPage = 10; 
+        let rows = document.querySelectorAll("#detail-html tbody tr");
+        
+        rows.forEach((row, index) => {
+            if (index % rowsPerPage === 0 && index !== 0) {
+                pageNumber++;
+            }
+            row.setAttribute("data-page", pageNumber);
+        });
+
+        document.getElementById("page-number").textContent = pageNumber;
```

Comment:
why you show page number on javascript na ? 

Please try with this code 
{{ _("Page {0}/{1}").format('<span class="page"></span>', '<span class="topage"></span>') }}